src/main/java/com/ioet/bpm/peopletime/domain/TimeEvent.java

```diff
@@ -0,0 +1,58 @@
+package com.ioet.bpm.peopletime.domain;
+
+import com.amazonaws.services.dynamodbv2.datamodeling.*;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.validation.constraints.NotBlank;
+import java.util.Date;
+
+@NoArgsConstructor
+@AllArgsConstructor
+@Builder
+@Data
+@DynamoDBTable(tableName = "time_event")
```

Comment:
The idea is to prepend a prefix for the tables. For example for the service `people` if we are creating the table `time`. The table full name should be:

people_time

Please add the suffix for this table